examples/acoustic/Acoustic_codegen.py

```diff
@@ -59,6 +61,7 @@ def __init__(self, model, data, source, nbpml=40, t_order=2, s_order=2,
             self.at = AutoTuner(fw)
             self.at.auto_tune_blocks(self.s_order + 1, self.s_order * 4 + 2)
 
+    # Forward modelling
```

Comment:
Python docstrings would be better, since documentation-generators pick them up. Same for the other classes below.